python/docs/source/conf.py

```diff
@@ -45,8 +45,19 @@
     'sphinx.ext.viewcode',
     'sphinx.ext.mathjax',
     'sphinx.ext.autosummary',
+    'nbsphinx',  # Converts Jupyter Notebook to reStructuredText files for Sphinx.
+    # For ipython directive in reStructuredText files. It is generated by the notebook.
+    'IPython.sphinxext.ipython_console_highlighting'
 ]
 
+# Links
```

Comment:
Can we add more few words explaining where these links locate? Like "Links used in python/docs/source/index.rst".

Reply:
Ah .. I thought I commented like that. Sure👌

Reply:
Ahh .. there were some changes not pushed into this PR. I will push soon. https://github.com/apache/spark/pull/29491#discussion_r474399231 too ..